tests/model_hub_tests/pytorch/requirements.txt

```diff
@@ -26,3 +26,34 @@ transformers
 wheel
 PyYAML
 kornia
+
+openvino-tokenizers
```

Comment:
please exclude and install tokenizers built in our GHA. See TF Hub model tests

Reply:
remoeved.

Reply:
still unremoved